.readme-partials.yml

```diff
@@ -80,20 +80,18 @@ body: |-
   If you already have a "raw" Http `request` object you can assign it to `entry.metadata.httpRequest` directly. More information about
   how the `request` is interpreted as raw can be found in the [code](https://github.com/googleapis/nodejs-logging/blob/15849160116a814ab71113138cb211c2e0c2d4b4/src/entry.ts#L224-L238).
 
-  ## Automatic Trace/Span Id Extraction
-  Cloud Logging libraries use [trace fields within LogEntry](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace) to capture trace contexts, which enables the [correlation of logs and traces](https://cloud.google.com/logging/docs/view/correlate-logs), and supporting distributed tracing. 
-  These trace fields, including [trace](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace), [spanId](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.span_id), and [traceSampled](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace_sampled) define the trace context for the log entry.
+  ## Automatic trace/span Id extraction
+  Cloud Logging libraries use [trace fields within LogEntry](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace) to capture trace contexts, which enables the [correlation of logs and traces](https://cloud.google.com/logging/docs/view/correlate-logs), and distributed tracing troubleshooting. 
+  These tracing fields including [trace](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace), [spanId](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.span_id), and [traceSampled](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace_sampled) define the trace context for a `LogEntry`.
 
-  If not provided explicitly, Google Cloud Logging library will automatically populate `LogEntry fields`_
-  `trace`, `span_id`, and `trace_sampled` from detected OpenTelemetry span context, or HTTP request headers.
+  If not provided explicitly in a LogEntry, Google Cloud Logging library will automatically populate `trace`, `span_id`, and `trace_sampled` fields from detected OpenTelemetry span contexts, or HTTP request headers.
```

Comment:
I'd drop "Google" (the official product name is "Cloud Logging") and subsitute "the", for "the Cloud Logging library"

I'd change "will automatically populate `trace`..." to "automatically populates the `trace` ..."

I'd change "span contexts, or HTTP..." to "span context or from HTTP..."